VetClinicPublic/src/VetClinicPublic.Contracts/AppointmentConfirmLinkClickedIntegrationEvent.cs

```diff
@@ -0,0 +1,22 @@
+namespace VetClinicPublic.Contracts;
+
+public class AppointmentConfirmLinkClickedIntegrationEvent
+{
+  public AppointmentConfirmLinkClickedIntegrationEvent(Guid appointmentId)
```

Comment:
But this one is still an integration event ?

Reply:
See comment above.